MmSupervisorPkg/Core/Relocate/Relocate.h

```diff
@@ -739,4 +739,15 @@ ProcessUserSaveStateAccess (
   IN UINT64               Arg3
   );
 
+/**
+  Function to perform post relocation logic before handing back to the IPL.
+
+  TODO: Inspect the page tables. The pages allocated in this SMI may not be protected.

```

Comment:
Might be good to track this todo with an issue in the repo.

Reply:
This should not be there anymore. The implementation was updated to keep the original page table configuration flow.